src/Monolog/Handler/Slack/SlackRecord.php

```diff
@@ -85,7 +91,7 @@ class SlackRecord
      */
     private $normalizerFormatter;
 
-    public function __construct($channel = null, $username = null, $useAttachment = true, $userIcon = null, $useShortAttachment = false, $includeContextAndExtra = false, array $excludeFields = array(), FormatterInterface $formatter = null)
+    public function __construct($channel = null, $username = null, $useAttachment = true, $userIcon = null, $useShortAttachment = false, $includeContextAndExtra = false, array $excludeFields = array(), array $markdownIn = array(), FormatterInterface $formatter = null)
```

Comment:
If you want this in a 1.x release the argument need to be added last otherwise it's a BC break.